src/modules/user/schema/user.schema.ts

```diff
@@ -38,11 +39,28 @@ export class User {
   @Prop()
   created_at: Date;
```

Comment:
Delete this field.
Instead add timeStamp to schema like this:  https://github.com/codeforcauseorg/edu-server/commit/b2c509c5c6329eb0789d25d903c72a2f4707a8e7#r51469101